eatngo-auth-common/src/main/resources/application.yml

```
@@ -0,0 +1,24 @@
+              - profile_nickname
+              - account_email
+            client-name: Kakao
+        provider:
+          kakao:
+            authorization-uri: https://kauth.kakao.com/oauth/authorize
+            token-uri: https://kauth.kakao.com/oauth/token
+            user-info-uri: https://kapi.kakao.com/v2/user/me
+            user-name-attribute: id
+jwt:
+  secret: YW5Zb3UtY2FuLXVzZS10aGlzLWtleS1mb3ItanVzdC10ZXN0aW5nIQ==
+  # anYou-can-use-this-key-for-just-testing! Base64-encoded -> // TODO 교체 및 외부 주입
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)